### ISSUE TITLE
Perspective Transformations

### ISSUE THREAD

**Issue by MentalGear** [aleju/imgaug#393]
Perspectie transformations seems to be available, but not listed at https://imgaug.readthedocs.io/en/latest/source/augmenters.html

**Comment by aleju** [comment]
Yes, that overview page is very out of date and misses like half of all augmenters...
There will be an updated one upon release of the next version of the library. It is just not merged yet, because the next version also contains some breaking changes.

**Comment by marz233** [comment]
When I using Perspective Transformation there is some problem:
The polygon after perspective transformation should stay in the image boundary,not like the image show below
![Screenshot from 2019-09-26 19-06-11](https://user-images.githubusercontent.com/10582496/65684030-3ffdfb00-e091-11e9-8754-afaf7b095c5b.png)
Is it a bug?  :P

**Comment by marz233** [comment]
emm... I got clip_out_of_image() to solve the problem above
psoi_aug=psoi_aug.clip_out_of_image()

**Comment by marz233** [comment]
I found clip_out_of_image() can only clip one polygon to another one,the problem is concave polygon after clip might divide into more than one polygon：P
How should I deal with this？

**Comment by aleju** [comment]
`Polygon.clip_out_of_image()` returns a list of 0..N polygons, i.e. one input polygon can become multiple polygons after clipping. Isn't that what you want?

**Comment by lamhoangtung** [comment]
> When I using Perspective Transformation there is some problem:
> The polygon after perspective transformation should stay in the image boundary,not like the image show below
> ![Screenshot from 2019-09-26 19-06-11](https://user-images.githubusercontent.com/10582496/65684030-3ffdfb00-e091-11e9-8754-afaf7b095c5b.png)
> Is it a bug exist?
> Maybe clipping algorithm like Sutherland-Hodgman or Weiler-Atherton can solve this problem？ :P

Check out this PR #452, it's already merged into `master`. So you just need to:
```bash
pip install git+https://github.com/aleju/imgaug
```
and do this:
```python
iaa.PerspectiveTransform(scale=(0.075, 0.1), fit_output=True, mode='constant', cval=0)
```
